src/Arcus.EventGrid.Publishing/Arcus.EventGrid.Publishing.csproj

```diff
@@ -19,7 +19,7 @@
     <PackageReference Include="Flurl.Http" Version="2.3.1" />
     <PackageReference Include="Guard.NET" Version="1.2.0" />
     <PackageReference Include="Newtonsoft.Json" Version="11.0.1" />
-    <PackageReference Include="Polly-Signed" Version="5.9.0" />
+    <PackageReference Include="Polly-Signed" Version="5.9.0" />	
```

Comment:
Unnecessary whitespace :)

Reply:
Removed them, think the remaining here are just a diff-thing.

![image](https://user-images.githubusercontent.com/4345663/47421977-f8386a80-d781-11e8-89cd-faee426b4621.png)